integration/e2e/encodings_test.go

```diff
@@ -0,0 +1,121 @@
+package e2e
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/grafana/e2e"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+
+	"github.com/grafana/tempo/cmd/tempo/app"
+	"github.com/grafana/tempo/integration"
+	"github.com/grafana/tempo/integration/e2e/backend"
+	"github.com/grafana/tempo/pkg/httpclient"
+	"github.com/grafana/tempo/pkg/util"
+	"github.com/grafana/tempo/tempodb/encoding"
+	v2 "github.com/grafana/tempo/tempodb/encoding/v2"
+)
+
+const (
+	configAllEncodings = "./config-encodings.tmpl.yaml"
+)
+
+func TestEncodings(t *testing.T) {
+	const repeatedSearchCount = 10
+
+	for _, enc := range encoding.AllEncodings() {
+		if enc.Version() == v2.VersionString {
```

Comment:
if it's possible to do the trace by id lookup only on v2 we should. no worries if not.

Reply:
Done!